src/bootstrap/src/core/config/config.rs

```diff
@@ -1072,7 +1065,7 @@ impl Config {
 
         let mut git = helpers::git(Some(&self.src));
         git.arg("show").arg(format!("{commit}:{}", file.to_str().unwrap()));
-        output(git.as_command_mut())
+        git.allow_failure().run_capture_stdout(self).stdout()
```

Comment:
This shouldn't have `allow_failure()`, because before (with `output`), failure of the command would panic, right?